### PR TITLE
Submit RWS for Schibsted

### PR DESCRIPTION
**Pull request by JakBialy**

### CODE REVIEW

**Comment by JakBialy** [comment]
Hello ;) Seems like it is waiting for review 😎

**Review by cfredric — APPROVED** [review]

**Comment by Parasos** [comment]
9717e9887acc53f9e3318e4644ab0088c9bcfde1

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

File: related_website_sets.JSON [hunk 1]
```diff
@@ -667,7 +667,23 @@
...
+      "contact": "rws-admin@schibsted.com",
+      "associatedSites": [
+        "https://prisjakt.no",
+        "https://mittanbud.no"
+      ],
+      "serviceSites": [
+        "https://pdmp-apis.no"
+      ],
+      "rationaleBySite": {
+        "https://prisjakt.no": "Site owned by Schibsted with shared SSO across Schibsted Norway's sites and common branding that clearly shows the relation between the sites.",
```
Comment (Parasos) [inline related_website_sets.JSON:681]: related_website_sets.JSONRwsCheck.py

**Pull request merged**